### PR TITLE
uasyncio.core: Prevent the event loop to be run twice

### PR DESCRIPTION
**Pull request by psclafer**
Following the discussion on PR #9 , this patch prevents the event loop to be run again in theses 2 cases:
- Loop stopped by StopLoop
- Loop stopped by an Exception

### CODE REVIEW

**Comment by pfalcon** [comment]
> Following the discussion on PR #9

Yeah, so the discussion in #9, which was preempted by its closure:

> I was not aware that the async loop was intended to be run only once. I understand now why do you think, that it's not mandatory to clean up properly.

I don't know why you get that idea, that's definitely not what I was saying. You somehow assume that there's the issue which oh-so-urgently needs to be fixed, in binarized "0" or "1" way ("binarized" is the only reason I can explain why you abruptly closed that PR, just to submit this).

But *not every issue needs to be fixed*. Let me explain that picturesquely: you go out of your city, walk by road, then by the woods, up the hill and suddenly ... a pit! Pits are dangerous, someone can fall into them! So you're rushing to build a fence around it. That will cost you a lot of money and effort, the same for those who'll cross your way and think the same that there's something to do about a random pit somewhere out there. But heck, nobody is even going there. And those who do, are smart enough to not get into it.

You also saw a one or two issues, and I as a maintainer see thousands of them. As you imagine, I wouldn't have a life if I rushed to fix all of them. So, all they have relative priorities and I'm interested to fix high-priority issues, medium-priority issues, and almost never low-priority issues. I'm interested in contributors to apply the same logic too (learn what priorities are, help adjust priorities, etc.).

Summing up: this issue can stay open. People who managed to get into a pit are welcome to share their stories here. Once an array of *convincing* stories accumulate, there will be something to fix.

**Comment by pfalcon** [comment]
No need for these changes arised over this time, closing.
